yt/chyt/controller/internal/api/helpers.go

```diff
@@ -92,6 +93,30 @@ func transformToStringSlice(value any) (any, error) {
 	return transformedAttributes, nil
 }
 
+func transformFilterValues(value any) (any, error) {
+	if value == nil {
+		return nil, nil
```

Comment:
`return nil` and `return map[string]any(nil)` are different in this context - the first one doesn't store the type info and `x.(map[string]any)` would cause panic (I'll fix it during the import)